src/AdvancedMH.jl

```diff
@@ -50,7 +50,7 @@ end
 
 # Create a very basic Transition type, only stores the
 # parameter draws and the log probability of the draw.
-struct Transition{T<:Union{Vector, Real, NamedTuple}, L<:Real} <: AbstractTransition
+struct Transition{T<:Union{AbstractVector, Real, NamedTuple}, L<:Real} <: AbstractTransition
```

Comment:
Is the type constraint useful? Or should we just define
```suggestion
struct Transition{T,L<:Real} <: AbstractTransition
```
?

Reply:
I thought the same thing just after making the suggestion. I'm very fine with removing it completely, but I'm not too familiar with internals of AMH.jl :shrug:

Reply:
So you / @cpfiffer make the call!

Reply:
Hm. I don't mind either way, honestly, which suggests to me that we should just remove the type constraint entirely.